### PR TITLE
chore: fix vue lint warnings

### PR DESCRIPTION
**Pull request by chmelevskij**
Fix linter warnings for vue components.

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]

File: src/components/status-bar/StatusBar.vue
Comment (Asizon) [inline src/components/status-bar/StatusBar.vue:63]: If we have defaults defined here, maybe we should remove these defaults in fc.js ??

File: src/components/status-bar/StatusBar.vue
Comment (Asizon) [inline src/components/status-bar/StatusBar.vue:67]: Same here and also for the others elements.

**Review by Asizon — CHANGES_REQUESTED** [review]

File: src/components/status-bar/StatusBar.vue
Comment (Asizon) [reply src/components/status-bar/StatusBar.vue:63]: But i think is good to continue with defaults in fc.js and remove these defaults

**Comment by Asizon** [comment]
@chmelevskij These changes are really needed for 10.8.0??

File: src/components/status-bar/StatusBar.vue
Comment (chmelevskij) [reply src/components/status-bar/StatusBar.vue:63]: I'd keep both, this is just for component, it shouldn't care what's in `fc.js` or other files. There is logic which relies on this values inside of it.

We already have everything to tightly coupled. So relying on things which are in some global variable in random file is a pain in the neck to maintain as it is.

File: src/components/status-bar/StatusBar.vue
Comment (chmelevskij) [reply src/components/status-bar/StatusBar.vue:67]: Same here, defaults here are for the component encapsulation, not for the global variables.

**Comment by chmelevskij** [comment]
Can wait, but would be good to get them going. They also are not new features so should be safe to merge.

I already spend a lot of time rebasing my stuff constantly, so would appreciate if we could either get that 10.8 out or just merge these, as I'm doing them in small chunks and not doing any functional changes really.

**Review by Asizon — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by limonspb — APPROVED** [review]

**Pull request merged**
